config/initializers/time_with_zone.rb

```diff
@@ -0,0 +1,5 @@
+class ActiveSupport::TimeWithZone
```

Comment:
Does this class change the configuration for everything to now just year/month/day?

Reply:
it makes it so when you call `to_json` on a datetime object, you only get the year/month/day